azure-pipelines.yml

```diff
@@ -64,4 +65,5 @@ jobs:
         git commit --allow-empty -m "Documentation for $(GitVersion.PreReleaseTag)"
         git push origin gh-pages
       displayName: 'Publish documentation to GitHub Pages'
-      condition: and(succeeded(), or(variables['Endjin.ForceDocPublish'], eq(variables['GitVersion.PreReleaseTag'], '')))
\ No newline at end of file
+      condition: and(succeeded(), or(variables['Endjin.ForceDocPublish'], eq(variables['GitVersion.PreReleaseTag'], '')))
+    compileTasksServiceConnection: endjin-acr-reader
```

Comment:
I think moving this line earlier in the file, before the customisations would make it a bit easier to read